src/app/globals.css

```
@@ -473,6 +473,198 @@ pre[data-code-block].is-collapsed {
+  gap: clamp(0.75rem, 2vw, 1.5rem);
+}
+
+.site-header__links a {
+  color: var(--text);
+  font-weight: 600;
+  text-decoration-color: transparent;
+}
+
+.site-header__links a:hover {
+  color: var(--link);
+  text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)